pkg/restore/backoff.go

```diff
@@ -67,7 +60,7 @@ func newDownloadSSTBackoffer() utils.Backoffer {
 
 func (bo *importerBackoffer) NextBackoff(err error) time.Duration {
 	switch errors.Cause(err) {
-	case errResp, errGrpc, errEpochNotMatch:
+	case errGrpc, errEpochNotMatch, errIngestFailed:
```

Comment:
Should we retry on`errRewriteRuleNotFound`? 

TiKV can return region not found after https://github.com/tikv/tikv/pull/7135

Reply:
`errRewriteRuleNotFound` is for handling with the region which exists in the range [t{tableid}, t{tableid+1}), but don't match to any rewrite rule, such as the region [t{tableid}, t{tableid}_i1). We should skip them.

Reply:
Oops, I mean `errRegionNotFound` (`errRewriteRuleNotFound` is a typo 😂

Reply:
Now `errRegionNotFound` is included in `errIngestFailed`